src/commands/subcommands/config/handleSet.ts

```diff
@@ -69,6 +69,19 @@ export const handleSet: SettingsHandler = async (
       }
     }
 
+    if (
+      setting === "level_roles" &&
+      interaction.options.getRole("role", true).position <
```

Comment:
```suggestion
      interaction.options.getRole("role", true).position >
```
Think this needs to be inverted. The everyone role is at the bottom of the list and has a position of 0. The higher the role in the list, the higher the position.

Reply:
oh